app/styles/user/certificates-view.ozar.sass

```diff
@@ -119,34 +142,158 @@
         line-height: 30pt
 
       &.code-results
-        left: 0.0in
+        @if $is-codecombat
+          left: 0.1in
+        @if $is-ozaria
+          left: 0.0in
 
       &.concepts-learned
-        left: 3.9in
+        @if $is-codecombat
+          left: 4in
+        @if $is-ozaria
+          left: 3.9in
 
       li
         text-transform: uppercase
         font-size: 9pt
-        line-height: 16pt
+        @if $is-codecombat
+          line-height: 18pt
+        @if $is-ozaria
+          line-height: 16pt
 
         a
           text-transform: none
 
-    .hero-signature
+    @if $is-codecombat
+      .hero-pose
+        position: absolute
+        bottom: 0.75in
+        z-index: 2
+        width: 1.7in
+        left: 0.8in
+
+        &.captain
+          transform: scaleX(-1)
+
+        &.knight
+          width: 2.5in
+          left: 0.3in
+
+        &.samurai
+          width: 1.7in
+          left: 0.8in
+
+        &.raider
+          width: 2in
+          left: 0.6in
+
+        &.goliath
+          width: 2.8in
+          left: 0in
+
+        &.guardian
+          width: 2.2in
+          left: 0.55in
+
+        &.duelist
+          width: 2.1in
+          left: 0.6in
+
+        &.champion
+          width: 1.7in
+          left: 0.8in
+
+        &.stalwart
+          width: 2.3in
+          left: 0.5in
+
+    .hero-signature, .product-codecombat & .teacher-info
       position: absolute
       bottom: 0.1in
       left: 0.6in
-      width: 1.8in
-      border-top: 2pt solid rgb(227, 195, 71)
+      @if $is-codecombat
+        width: 3in
+        border-top: 2pt solid white
+      @if $is-ozaria
+        width: 1.8in
+        border-top: 2pt solid rgb(227, 195, 71)
 
       img
-        height: 0.3in
-        position: absolute
-        bottom: 0.3in
-        left: 0
+        @if $is-codecombat
+          height: 0.75in
+          position: absolute
+          bottom: 0
+          left: -0.2in
+        @if $is-ozaria
+          height: 0.3in
+          position: absolute
+          bottom: 0.3in
+          left: 0
 
       .hero-signature-label
         text-transform: uppercase
         font-size: 9pt
         line-height: 18pt
         text-align: left
+
+    @if $is-codecombat
+      .product-codecombat & .teacher-info
```

Comment:
have we added this product-codecombat class somewhere or it's getting added automatically?

Reply:
@mrfinch its added in CocoView constructor: https://github.com/codecombat/codecombat/blob/master/app/views/core/CocoView.coffee#L62